solutions/src/test/java/ru/sberbank/school/task08/SerializableManagerTest.java

```diff
@@ -0,0 +1,61 @@
+package ru.sberbank.school.task08;
+
+import org.junit.Assert;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import org.junit.Before;
+import org.junit.Test;
+import ru.sberbank.school.task08.state.Savable;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SerializableManagerTest {
+    private Savable mapState;
+    private SerializableManager manager;
+    private String path;
+    private String fileName;
+
+    public SerializableManagerTest () {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1303446116>*